src/DevBetterWeb.Web/Services/VideoDetailsService.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Threading.Tasks;
+using Ardalis.ApiClient;
+using DevBetterWeb.Core.Entities;
+using DevBetterWeb.Core.Interfaces;
+using DevBetterWeb.Core.Specs;
+using DevBetterWeb.Infrastructure.Identity.Data;
+using DevBetterWeb.Vimeo.Models;
+using DevBetterWeb.Vimeo.Services.VideoServices;
+using Microsoft.AspNetCore.Identity;
+
+namespace DevBetterWeb.Web.Services;
```

Comment:
I think this service needs to be in the Core project.

Reply:
Because this service depends on the Infrastructure and Vimeo projects, I will keep it in the  Web project for the moment.